src/begateway.js

```diff
@@ -45,10 +45,21 @@
     }
   }
 
+  BeGateway.prototype.showSpinner = function(){
+    $('#' + this.anchor).append(
+      "<div class='spinner'><img src='https://js.begateway.com/images/ajax-loader.gif' class='beSpinner'></img></div>");
```

Comment:
I think no, we are loading this image before iFrame + if we stick to iFrame url, iFrame may be hosted on different domains.